includes/definitions/discovery/fxm.yaml

```diff
@@ -2,6 +2,8 @@ mib: mibs/alpha/Argus-Power-System-MIB
 modules:
     sensors:
         voltage:
+            options:
+              divisor: 10
```

Comment:
So this is the only thing affecting graphs?

Reply:
Yes, all three appear to need the divisor.

Reply:
Ok but the PR states it fixes current, voltage and frequency but I can only see voltage as being the one that is actually changed.

Reply:
Changed the title to correspond with *just* voltage being changed.